tests/crypto/rng-cc310.overlay

```diff
@@ -0,0 +1,5 @@
+/ {
+    chosen {
+        zephyr,entropy = &cryptocell;
+    };
+};
```

Comment:
I assume this is not needed when this fix gets in:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/2261

Could you try this PR with the fix in PR #2261 ?

Reply:
After a visual inspection:
I agree this won't be necessary.

But if cryptocell is default, I could flip this around and have an overlay for `&rng` instead.
It might be useful in the `overlay-sw` case to do "everything" in SW.

Thoughts?

Reply:
Would it ?
If we have already decided that CC should be used whenever it is available, then efforts might be better spent elsewhere.
After all, the tests for nrf52 (except nrf52840) will already be testing the `&rng`.

Reply:
Fair point.
Tried out the west.yml in #2261, worked nicely without any overlays.

This patch then only introduces a forgotten CONFIG_ENTROPY_NAME change.